src/Components/ServiceScene/LogsPanelScene.tsx

```diff
@@ -310,8 +314,38 @@ export class LogsPanelScene extends SceneObjectBase<LogsPanelSceneState> {
       .setHeaderActions(
         new LogOptionsScene({ visualizationType, onChangeVisualizationType: parentModel.setVisualizationType })
       )
-      .build();
-  }
+      .setOption('sortOrder', this.state.sortOrder)
+      .setOption('wrapLogMessage', this.state.wrapLogMessage)
+      .setOption('prettifyLogMessage', this.state.prettifyLogMessage);
+
+    if (!logsControlsSupported) {
+      panel.setOption('showTime', true);
+    } else {
+      panel
+        .setOption('showTime', getBooleanLogOption('showTime', true))
+        // @ts-expect-error Requires Grafana 12.1
```

Comment:
lets create an issue to remove these when our min grafana version is >=12.1

Reply:
Done.